Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[workspace]
+members = [
+    "workflow-types",
+    "workflows",
```

Comment:
we need a cargo workspace here because we need to import the Workflow type definitions within the workflows builds.rs script, which means it needs to be defined within another crate